test/types/schema.test.ts

```diff
@@ -374,12 +374,12 @@ export function autoTypedSchema() {
     boolean2?: boolean;
     boolean3?: boolean;
     boolean4?: boolean;
-    mixed1?: any;
-    mixed2?: any;
-    mixed3?: any;
-    objectId1?: Schema.Types.ObjectId;
-    objectId2?: Schema.Types.ObjectId;
-    objectId3?: Schema.Types.ObjectId;
+    mixed1?: unknown;
+    mixed2?: unknown;
+    mixed3?: unknown;
+    objectId1?: Types.ObjectId;
```

Comment:
This is correct according to the docs. "Types.ObjectId" instead of "Schema.Types.ObjectId"